script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_1B.txt

```diff
@@ -212,14 +212,14 @@ So, was I really the only one you wanted to kill?"
 "...Yeah. I've never felt that way about anyone else but you."
 <Page67>
 "Well then, in that case there's no problem.
-Come on, lift your head, Shiki. You are nothing close to a murderer."
+Come on, lift your head, Shiki. You are nothing like a murderer."
```

Comment:
This is kinda a mistranslation (and doesn't super make sense as-is).
> あなたは殺人鬼なんかじゃないよ
You're not some bloodthirsty killer.